locale/core.yml

```diff
@@ -131,6 +131,10 @@ core:
       open_modal: Open Settings
       permissions_title: Permissions
       uninstall_button: Uninstall
+      readme:
```

Comment:
`readme` should be sorted alphabetically in the keys here (directly below `permissions_title`).

The keys inside `readme` should also be sorted alphabetically.